### DISCUSSION TITLE
Add Member AJAX undefined person

### DISCUSSION THREAD

**Discussion by Sp0run** [liedekef/events-made-easy#641] [General]
There looks to be a buglet with the ajax for finding people on the add member admin form. I am trying to transfer a membership to another person but no matter what I type in the person text box, the typeahead returns undefined and clears the text box. It also appears to be the same with transferring 'head of accounts'

**Comment by Sp0run** [comment]
![image](https://github.com/user-attachments/assets/8df45c45-909c-42e7-a8fa-e30efdb5c489)

**Comment by liedekef** [comment]
which version of EME?
Concerning "head of family": there was indeed a typo in the related javascript file, this fixes it (and a php warning): https://github.com/liedekef/events-made-easy/commit/c75df79ad549c893e82162e8b5e03f4af1537b79

**Comment by Sp0run** [comment]
Sorry! version 2.5.9 thanks for the speedy head of family fix!
